services/apps/test/apps/persistence/categories_test.clj

```diff
@@ -1,7 +1,14 @@
 (ns apps.persistence.categories-test
   (:use [clojure.test]
         [apps.persistence.categories]
-        [apps.test-db-setup]))
+        [apps.test-fixtures :only [run-integration-tests with-test-db]]
+        [korma.core]))
+
+(defn clean-up-hierarchy-versions [f]
+  (f)
+  (delete :app_hierarchy_version))
+
+(use-fixtures :each with-test-db run-integration-tests clean-up-hierarchy-versions)
```

Comment:
That's true, as long as the cleanup is done after the tests:

```clojure
(use-fixtures :once run-integration-tests with-test-db clean-up-hierarchy-versions)
```

That's roughly equivalent to the code as it is right now, though, because there's only one test. The primary reason I made this change was so that I can run `lein test` repeatedly without having to reinitialize the database; it helps me to make my testing cycle faster.